.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
+node_js:
+- node
+script:
+- npm run test
+deploy:
+  provider: npm
+  email: 'systems@azavea.com'
+  api_key: ''
+  env:
+    global:
+      secure: qkinmGg7embShBpjy9P4617XlcvpmgThBiO5NfGVoX5ciPshhuAc1je0wMyI4SiPElibq/8AS/L6Qa8pyPG/RYeo4zg4ZRFoMvpffKlmvBhx1Y+kEpfL3GBLKNb7MS6nHq/1YlqtJ0JU/REVjgn1wMdLWLaNEoeP0VoGp6CbGl+IsdqxcepPDQ1cZ4mHnlCDhkbRqpyEyTjV/Z77/ZXpHrWvb1WkInuYP7BxX1t0vTGsTZpW9kGLHU+h6XRoXTJsbZLlZUiVtsWGJypjo8RcagH0bIi8btiGQdTKjboHDjHrt04eciF7LgsmEUll2Go/v6UxQRKHP+l4AmxQ+nhN1P8ZjwnXWMlUaQls86BkieYiM3lt4SZPQpctPGa5aDa93OSb7TFzYG82mFUrIbBAtmenw2aY6GOxEWs3/k3TiMR0KnYHRZ67sLVmN6My7wgCxcpVuWS3GVPbBXj8TbL+tgdiWAap/xj+jSUrGu9ECq645+8U36UeC2HK/mvJ17TAKgHmJ06m5qXvq4IziIAQCHicZxTANorgfq1Ok/mRPiLRAqz9inMK3E/oKihj48ibXxhE5ucS8mnMRHewVNeyJRbu+D/OKElA0BEBQdQWmanv3t2WTzOR/9qzeNhXw5Zpw6lr5j2a++JpbrtxKbrv9nOmBKjLIWSmWLPK2cBqlII=
+  on:
+    tags: true
+    repo: azavea/tilejson-validator
+    branch: master
```

Comment:
I don't think specifying `branch` here is desirable and looks like it's overridden by `tags`:

https://docs.travis-ci.com/user/deployment#Conditional-Releases-with-on%3A

We want this to release as part of the git flow release process, which uses tags to identify the release.